mdslib/MdsLib.c

```diff
@@ -1192,8 +1192,7 @@ static int mds_put2_vargs(va_list incrmtr, int connection, char *pathname,
       if (STATUS_OK)
       {
         if ((status =
-                 TreePutRecord(nid, (struct descriptor *)arglist[argidx - 2]),
-             0) &
+                 TreePutRecord(nid, (struct descriptor *)arglist[argidx - 2], 0)) &
```

Comment:
Yes, we could put the trailing `1)` back on this line.   I didn't do that simply because the original `mds_put_vargs()` was also mangled.   I figured that preserving the mangled structure made it easier to diff the two functions (as might be needed for future maintenance work).

I'm happy though to move the trailing `1` up to line 1195 if you wish.   Additional thoughts?

Reply:
If they're both mangled, I would unmangle them both, but either way is fine

Reply:
OK, I will unmangle them both.

Reply:
Hmm . . . Just occurred to me that you likely wanted `status = TreePutRecord . . . ` all on one line.   If so, let me know and I'll make another commit.

Reply:
Yeah, that would probably be best

Reply:
OK, will do.